configs/det/dbnet/db_r50_icdar15.yaml

```diff
@@ -148,3 +148,41 @@ eval:
     batch_size: 1 # TODO: due to dynamic shape of polygons (num of boxes varies), BS has to be 1
     drop_remainder: False
     num_workers: 2
+
+predict:
```

Comment:
Because of the separate `PredictDataset` class, there is many duplicates in the config file. It's easy to make a mistake when there's a lot of repetitions.